.eslintrc.js

```diff
@@ -33,6 +33,7 @@ module.exports = {
     'import/no-namespace': 'off',
     'import/no-named-export': 'off',
     'no-unused-vars': 'off',
+    'spaced-comment': 'off',
```

Comment:
Is it really necessary to mess with global linting rules? If there is really a good reason to ignore it at some point, rather use a `// eslint-disable-next-line spaced-comment`.

Reply:
There seems to be a bug with this rule and '.d.ts' files. I tried a lot of things to make it work, including

```
/* eslint-disable spaced-comment */
```
in the file but it seems to ignore it :/

Whenever it tries to 'fix' the lint errors it ends up inserting random spaces everywhere and then still fails for a different reason.

Reply:
If it only affects `d.ts` files, then I would recommend adding it as an `override`:

```
overrides: [
  files: [**/*.d.ts],
  rules: [{'spaced-comment' : 'off'}]
]
```